ssl/pqueue.c

```diff
@@ -141,7 +141,7 @@ pitem *pqueue_next(pitem **item)
     return ret;
 }
 
-int pqueue_size(pqueue *pq)
+size_t pqueue_size(pqueue *pq)
 {
     pitem *item = pq->items;
     int count = 0;
```

Comment:
Any reason why the internal representation count stays as an int when the function interface has changed to a size_t? I assumed this would change internally as well?